app/code/community/Sendsmaily/Sync/Block/Rss.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/**
+ * Sendsmaily Sync
+ * Module to export Magento newsletter subscribers to Sendsmaily
+ * Copyright (C) 2010 Sendsmaily
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+class Sendsmaily_Sync_Block_Rss extends Mage_Rss_Block_Catalog_Abstract
+{
+    const CACHE_TAG = 'block_html_sync_rss_index';
+
+    protected function _construct()
+    {
+        // Setting cache to save the RSS for 5 minutes.
+        $this->setCacheTags(array(self::CACHE_TAG));
+        $this->setCacheKey('sendsmaily_rss_index');
+        $this->setCacheLifetime(300);
```

Comment:
Added RSS-feed caching. For 5 minutes the RSS-feed is returned from Magento cache, the `_toHtml()` function won't run.

Reply:
Can user flush the RSS cache from Magento admin panel?

Reply:
Yes, and if the caching is disabled, this caching function has no effect.
![image](https://user-images.githubusercontent.com/22918359/69227426-73896d00-0b8a-11ea-9a3b-217468e33638.png)

Reply:
Cool.